aws-lc-rs/src/encoding.rs

```diff
@@ -82,3 +84,12 @@ pub trait AsBigEndian<T> {
     /// Returns Unspecified if serialization fails.
     fn as_be_bytes(&self) -> Result<T, crate::error::Unspecified>;
 }
+
+/// Trait for values that can be serialized into a raw format
+pub trait AsRawBytes<T> {
```

Comment:
I seem to recall for ML-KEM we did this with the [`AsBigEndian`](https://docs.rs/aws-lc-rs/latest/aws_lc_rs/encoding/trait.AsBigEndian.html) trait?

Reply:
Since the value encoded for MLDSA is not technically "big endian" (which seems most apt to integer encodings), the use of the `AsBigEndian` trait for this seemed wrong. I thought an `AsRawBytes` trait would be most appropriate and likely to be re-used in the future.